.github/workflows/lower-bound-requirements.yml

```diff
@@ -18,17 +18,21 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+
     - name: Install dependencies and force lowest bound
       run: |
         python -m pip install --upgrade pip setuptools wheel
         python -m pip --no-cache-dir --quiet install --requirement lower-bound-requirements.txt
         python -m pip --no-cache-dir --quiet install --editable .[test]
         python -m pip install --requirement lower-bound-requirements.txt
         python -m pip list
+
     - name: Test with pytest
       run: |
-        python -m pytest -r sx --ignore tests/benchmarks/ --ignore tests/contrib --ignore tests/test_notebooks.py
+        # Run on tests/ to skip doctests of src given examples are for latest APIs
+        python -m pytest -r sx --ignore tests/benchmarks/ --ignore tests/contrib --ignore tests/test_notebooks.py tests/
```

Comment:
@kratsg I'm not sure if there is an easier way to do this that running explicitly over `tests/`.

Is there a way at the command line to negate the `--doctest-modules` from

https://github.com/scikit-hep/pyhf/blob/173c3c711c8037928a966d381c922fd535be956f/pyproject.toml#L44

?

Reply:
Nope

Reply:
Okay, cool. This is ready for review then. :)